Bent types: Cannot find name 'Response'

Attempting to compile a ts project, importing `@types/bent` (version 7.0.1) - getting compile error:

```
node_modules/@types/bent/index.d.ts:50:26 - error TS2304: Cannot find name 'Response'.
50     type FetchResponse = Response & { statusCode: number };
```

@Ovyerus, @elibarzilay - any ideas?

--- a/types/bent/index.d.ts
+++ b/types/bent/index.d.ts
@@ -5,6 +5,7 @@
 // TypeScript Version: 2.7
 
 /// <reference types="node" />
+/// <reference lib="dom" />
 
 import { PassThrough, Stream } from 'stream';
 

--- a/types/bent/index.d.ts
+++ b/types/bent/index.d.ts
@@ -2,7 +2,7 @@
 // Project: https://github.com/mikeal/bent#readme
 // Definitions by: Ovyerus <https://github.com/Ovyerus>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.7
+// TypeScript Version: 3.0
 
 /// <reference types="node" />
 /// <reference lib="dom" />

